tests/basic/legacy_shared_sharedv4_service_migration_tests.go

```diff
@@ -0,0 +1,655 @@
+package tests
+
+import (
+	"fmt"
+	"time"
+	"math/rand"
+
+	"github.com/portworx/torpedo/pkg/log"
+	"github.com/portworx/torpedo/pkg/testrailuttils"
+
+	"github.com/libopenstorage/openstorage/api"
+	"github.com/portworx/sched-ops/k8s/core"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/drivers/volume"
+	"k8s.io/apimachinery/pkg/types"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/portworx/torpedo/tests"
+)
+
+const (
+	ubuntu = "ubuntu-app"
+)
+
+// Legacy Shared Volume Create
+// Automatically it should get created as Sharedv4 service volume.
+
+var _ = Describe("{LegacySharedVolumeCreate}", func() {
+	var testrailID = 296369
+	// https://portworx.testrail.net/index.php?/cases/view/296369
+
+	StartTorpedoTest("LegacySharedVolumeAppCreateVolume", "Legacy Shared to Sharedv4 Service CreateVolume", nil, testrailID)
```

Comment:
u need to put this in JustBeforeEach else it will run even before any of the test start